cpp/include/cudf/io/parquet.hpp

```diff
@@ -95,12 +92,16 @@ class parquet_reader_options {
 
   /**
    * @brief Returns source info.
+   *
+   * @return Source info
    */
   [[nodiscard]] source_info const& get_source() const { return _source; }
 
   /**
    * @brief Returns true/false depending on whether strings should be converted to categories or
    * not.
+   *
+   * @return True/false depending on whether strings should be converted to categories or not.
```

Comment:
```suggestion
   * @return `true` if strings should be converted to categories.
```
Would this be easier to follow? The same for other boolean returns.

Reply:
will update this.